operator/pkg/ciliumendpointslice/ceptocesmap.go

```diff
@@ -167,6 +167,10 @@ func NewCESName(name string) CESName {
 	return CESName(resource.Key{Name: name})
 }
 
+func NewCESNameNamespace(name string, namespace string) CESName {
```

Comment:
```suggestion
func NewCESNamespacedName(name string, namespace string) CESName {
```
small nit based on the convention used in apimachinery and controller-runtime

Also, should its return be a new type to distinguish from CESName objects that don't have a namespace? Do we need to keep both or can we just use namespaced CES now instead?

Reply:
Thanks, I changed the name. There are still places, where NewCESName is used and namespace information is not needed (I believe that's the case of cesStore in reconciler). I did not modify it since it didn't concern the double queue.